apollo-router/src/layers/hello.rs

```diff
@@ -0,0 +1,47 @@
+use apollo_router_core::register_layer;
```

Comment:
would this be moved to some sorts of example folder eventually ?

Reply:
Yeah, we should get this set up.